trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -468,6 +471,10 @@ def make_spectrum(self, input_object, output_file=None,
         input_ds.domain_left_edge = input_ds.domain_left_edge.to('code_length')
         input_ds.domain_right_edge = input_ds.domain_right_edge.to('code_length')
 
+        self.h0 = getattr(input_ds,'hubble_constant')
```

Comment:
Can you change this to `hubble_constant` to be consistent with Trident/yt? I most frequently see H0 describing the Hubble constant in km/s/Mpc, whereas this one is little h.